tests/ClientTest.php

```diff
@@ -107,6 +107,42 @@ public function testCaptureException(): void
         $client->captureException($exception);
     }
 
+    /**
+     * @dataProvider captureExceptionWithExcludedExceptionsDataProvider
+     */
+    public function testCaptureExceptionWithExcludedExceptions(bool $shouldCapture, string $excluded, \Throwable $thrown): void
+    {
+        $transport = $this->createMock(TransportInterface::class);
+
+        $transport->expects($shouldCapture ? $this->once() : $this->never())
+            ->method('send')
+            ->with($this->callback(function (Event $event) use ($thrown): bool {
+                $this->assertCount(1, $event->getExceptions());
+
+                $exceptionData = $event->getExceptions()[0];
+
+                $this->assertSame(\get_class($thrown), $exceptionData['type']);
+                $this->assertSame($thrown->getMessage(), $exceptionData['value']);
+
+                return true;
+            }));
+
+        $client = ClientBuilder::create(['excluded_exceptions' => [$excluded]])
+            ->setTransport($transport)
+            ->getClient();
+
+        $client->captureException($thrown);
+    }
+
+    public function captureExceptionWithExcludedExceptionsDataProvider(): array
+    {
+        return [
+            [true, \Exception::class, new \Error()],
```

Comment:
Can we split this array into multiple lines (one for each item)? Also apply this fix to the rest of the items of the parent array